missile.pde

```diff
@@ -42,12 +43,21 @@ class Missile {
     vel = new PVector(0,0);
     accel = new PVector(0,0);
     parent = ship;
+    missileExplosion = new MissileExplosion(this);
   }
   
   PVector getMissilePos() {
     return pos;
   }
   
+  PVector getMissileVel() {
```

Comment:
I don't like the spacing on this, next time do better because I'm a mean co-worker

Reply:
well officer krupke, krup you!